core/src/components/side-menu/readme.md

```diff
@@ -21,6 +21,17 @@
 | `tdsCollapse` | Event that is emitted when the Side Menu is collapsed. | `CustomEvent<{ collapsed: boolean; }>` |
 
 
+## Slots
+
+| Slot             | Description                                                                            |
+| ---------------- | -------------------------------------------------------------------------------------- |
+| `"UNNAMED-SLOT"` | Used for nesting upper/top content in Side menu                                        |
```

Comment:
I think this is a great question. But my feeling is that these should not be named at all. The should remain as unnamed slots, and in the readme we either don't include it for now. Or we document it as unnamed/default slot.

Reply:
I think the reason for not including it in the readme would be that just adding an item, without a slot name is the expected behaviour. So to follow convention I guess this shouldn't be documented

Reply:
This is how liquid handles it, so the unnamed slot it just blank. 

<img width="569" alt="Screenshot 2023-07-03 at 10 23 14" src="https://github.com/scania-digital-design-system/tegel/assets/62651103/3429a289-ff02-4faa-9f84-c60cb95f368a">